tests/conftest.py

```diff
@@ -124,4 +124,3 @@ def check_output(xxx_todo_changeme):
 def detect_fails(capsys):
     yield capsys
     check_output(capsys.readouterr())
-    check_output(capsys._outerr)
```

Comment:
Sure?

Reply:
As said python, '_outerr' isn't a member of 'capsys'.